src/Cake.Common/Tools/OpenCover/OpenCoverHideSkippedOption.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Cake.Common.Tools.OpenCover
+{
+    using System;
+
+    /// <summary>
+    /// Represents the hide skipped option for OpenCover.
+    /// </summary>
+    [Flags]
+    public enum OpenCoverHideSkippedOption
+    {
+        /// <summary>
+        /// Removes no information from output file.
+        /// </summary>
+        None = 1,
```

Comment:
Just nitpicking ;) But I think in a flags scenario `None` has usually the value 0.

Reply:
Updated.